Documentation/how-to-build-ObjectWriter.md

```diff
@@ -0,0 +1,40 @@
+# Build ObjectWriter library #
+
+ObjWriter is based on LLVM, so it requires recent CMake and GCC/Clang to build LLVM.
+See http://llvm.org/docs/GettingStarted.html#requirements for more details.
+
+1. Clone LLVM from official LLVM mirror github git repository:
+
+    ```
+    $ git clone -b release_50 https://github.com/llvm-mirror/llvm.git
+    ```
+
+2. Copy ObjWriter directory from CoreRT into LLVM tree
+
+    ```
+    $ cp -r CoreRT/src/Native/ObjWriter llvm/tools/
+    ```
+
+3. Apply the patch to LLVM:
+
+    ```
+    $ cd llvm
+    $ git apply tools/ObjWriter/llvm.patch
+    ```
+
+4. Configure and build LLVM with ObjWriter:
+
+    ```
+    $ mkdir build
```

Comment:
It would be nice to omit the dollar signs at the start of the lines so that the blocks are easier to copy&paste.